lti/lti-common/src/java/org/sakaiproject/lti/util/SakaiLTIUtil.java

```diff
@@ -2650,6 +2655,7 @@ public static Object handleGradebookLTI13(Site site,  Long tool_id, Map<String,
 		String title;
 
 		log.debug("siteid: {} tool_id: {} lineitem_key: {} userId: {} scoreObj: {}", site.getId(), tool_id, lineitem_key, userId, scoreObj);
+System.out.println("content="+content);
```

Comment:
Is this println intended? If so, I recommend folding the content var into the log.debug line above.